school-library/persons.rb

```diff
@@ -1,14 +1,15 @@
 require './corrector'
 
 class Person
-  attr_reader :id, :parent_permission
+  attr_reader :id, :parent_permission, :rental
   attr_accessor :name, :age
 
   def initialize(age, name = 'unknown', parent_permission: true)
     @id = Random.rand(1..1000)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rental []
   end
 
   def _is_of_age?
```

Comment:
e.g. the Person class might have an `add_rental` method that takes a book instance and creates a new Rental. The same goes for the Book class. This way we will have multiple options to add_rentals in the future. Either through instances of the person, the book, or just calling a new rental

Reply:
- [ ] As the previous reviewer mentioned `add_rental` method should take a book instance and date to create a new rental and remember `The person` itself should be sent as a parameter to create `the new rental` achieving the 'has-many' association you can simply do 
```ruby
def add_rental(book, date)
    Rental.new(date, book, self)
  end
```
this applies as well to the `Book` class as well

Reply:
@DeliceLydia, thanks for the review but i'm having difficulty understanding you could you please explain